README.md

```diff
@@ -77,7 +77,7 @@
 
 - [Live Demo Link](https://chandan-devs-tech.github.io/Portfolio/)
 
-<img src="Portfolio-Finish-Mobile-Version_(iPhone SE).png" alt="Portfolio-Finish-Mobile-Version_(iPhone SE)" width="300px"  height="auto" />
+<img src="Portfolio-Mobile-Version(contact-form).png" alt="Portfolio-Mobile-Version" width="300px"  height="auto" />
```

Comment:
- [ ] Also to make your README more professional 😃 , kindly, make sure you added some information about partners because they contributed to this milestone 👍